conda/libmesh-vtk/build.sh

```diff
@@ -27,41 +12,24 @@ cmake .. -G "Ninja" \
     -DCMAKE_PREFIX_PATH:PATH=${VTK_PREFIX} \
     -DCMAKE_INSTALL_PREFIX:PATH=${VTK_PREFIX} \
     -DCMAKE_INSTALL_RPATH:PATH=${VTK_PREFIX}/lib \
-    -DCMAKE_INSTALL_LIBDIR=lib \
```

Comment:
In your commit message changing the libmesh-vtk package tests in meta.yaml, you wondered why lib needed to be changed to lib64. I think this is why. Why did you remove this?